webapp/actions/global_actions.jsx

```diff
@@ -534,3 +538,33 @@ export function emitBrowserFocus(focus) {
         focus
     });
 }
+
+export function redirectUserToDefaultTeam() {
+    const teams = TeamStore.getAll();
+    const teamMembers = TeamStore.getMyTeamMembers();
+    let teamId = BrowserStore.getGlobalItem('team');
+
+    if (!teamId && teamMembers.length > 0) {
+        let myTeams = [];
```

Comment:
We could discuss this more, but I don't think it's worth it to put in the extra work to pick the user's first team alphabetically as opposed to just picking a team randomly on the first load, especially if we store this across all devices. By going alphabetically, there's an equal chance that we'll get the "correct" team as if we were to pick randomly. I think this was before you were with us, but this is the same reason that new users get 10 random users in their DM channels when they create their account instead of the first 10 alphabetically.

Alternatively, I don't think there's an issue with just showing the Select Team page if you don't have a "previous team" stored anywhere since it'll only happen once and then it'll be saved after that.